docs/releases/1.7-calico-cidr-migration.md

```diff
@@ -0,0 +1,70 @@
+# Calico Pod CIDR Migration Procedure
+Prior to kops 1.7, calico was misconfigured to use the `.NonMasqueradeCIDR`
+field as the CIDR range for Pod IPs. As a result, IP conflict may occur
+when a Service is allocated an IP that has already been assigned to a Pod,
+or vice versa. To prevent this from occuring, manual steps are necessary before
+upgrading your cluster using kops 1.7 onwards.
+
+
+## Background
+The field in the clusterSpec, `.NonMasqueradeCIDR`, captures the IP
+range of the cluster.
+
+Within this IP range, smaller IP ranges are then carved out for:
+* Service IPs - as defined as `.serviceClusterIPRange`
+* Pod IPs - as defined as `.kubeControllerManager.clusterCIDR`
```

Comment:
Do we have an issue open for this?  We need to get a `.clusterCIDR` top level object.